hack/update-all.sh

```diff
@@ -23,9 +23,13 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/cluster/kube-env.sh"
 
 SILENT=true
+ALL=0
```

Comment:
(sorry for so many comments on so small a change)
for consistency, maybe use true/false as was done with SILENT?